.github/workflows/analyze_codebase.yml

```
@@ -0,0 +1,41 @@
+name: Analyze Codebase
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest  
+
+    steps:
+        - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)